docs/first-steps/management-account.md

```diff
@@ -0,0 +1,152 @@
+# Orchestrate the Management account
+
+Finally we reach the point in which you'll get to actually create the infrastructure in our AWS environment.
+
+Some accounts and layers rely on other accounts/layers being already deployed, creating dependencies between each other and establishing an order in which all layers should be deployed. We will go through these dependency chains in order.
+
+!!! success "Basic Landing Zone AWS Expenses"
+    By default this AWS Reference Architecture configuration should not incur in any costs.
+
+The **management** account is used to configure and access all AWS Organizations managed accounts, also, billing and financial decisions are enforced though this account.
+
+## Deploy the Management account's layers
+
+To begin, place yourself in the `management` account directory.
+
+``` bash
+cd management
+```
+
+### Terraform backend layer
```

Comment:
In this part we should add a link to an article that explains what is a Terraform Backend and why is it a good idea to use one. For instance:
- https://www.terraform.io/docs/language/settings/backends/index.html
- https://blog.gruntwork.io/how-to-manage-terraform-state-28f5697e68fa

Reply:
🤝✅